build.gradle

```diff
@@ -19,18 +19,35 @@ allprojects {
 
 dependencies {
     compile gradleApi()
-    compile 'org.codehaus.groovy:groovy-all:2.3.11'
+    compile 'org.codehaus.groovy:groovy-all:2.3.6'
```

Comment:
There was conflicting versions of the groovy-all dependency so had to down grade it